libsrc/telestrat/kbhit.s

```diff
@@ -6,12 +6,21 @@
 
         .export         _kbhit
 
+        .export         store_char
+
         .include        "telestrat.inc"
 
 _kbhit:
         BRK_TELEMON XRD0
```

Comment:
It's possible that `kbhit()` could be called several times before `cgetc()` is called.  Therefore, `kbhit()` should not call Telemon if a character already is waiting.